xray/audit/java/gradle.go

```diff
@@ -40,10 +48,45 @@ func runGradle(buildConfiguration *utils.BuildConfiguration, excludeTestDeps, us
 			log.Debug("Using resolver config from", configFilePath)
 		}
 	}
+	// check if gradle wrapper exist
+	if useWrapper {
+		var wrapperExist bool
+		wrapperExist, err = verifyGradleWrapper()
+		if err != nil {
+			return
+		}
+		useWrapper = wrapperExist
+	}
 	// Read config
 	vConfig, err := utils.ReadGradleConfig(configFilePath, useWrapper)
 	if err != nil {
 		return err
 	}
 	return gradleutils.RunGradle(vConfig, tasks, "", buildConfiguration, 0, useWrapper, true)
 }
+
+func verifyGradleWrapper() (bool, error) {
+	wd, err := os.Getwd()
+	if err != nil {
+		return false, err
+	}
+	filesInDir, err := fileutils.ListFiles(wd, false)
+	if err != nil {
+		return false, err
+	}
+	fullPathGradlew, err := filepath.Abs(gradlew)
+	if err != nil {
+		return false, err
+	}
+	fullPathGradlewbat, err := filepath.Abs(gradlewbat)
+	if err != nil {
+		return false, err
+	}
+	for _, file := range filesInDir {
+		if file == fullPathGradlew || file == fullPathGradlewbat {
+			return true, nil
+		}
+	}
+
+	return false, nil
```

Comment:
Please consider the following:
```suggestion
	wrapperName := gradlew
	if coreutils.IsWindows() {
		wrapperName += ".bat"
	}
	return fileutils.IsFileExists(wrapperName, false)
}
```

Reply:
done